handler/api/repos/builds/retry.go

```diff
@@ -64,6 +64,7 @@ func HandleRetry(
 		}
 
 		hook := &core.Hook{
+			Parent:       prev.ID,
```

Comment:
this should be `prev.Number`

Reply:
great catch, i had a mind fart obviously on friday afternoon.
Made the changes